zapcore/memory_encoder.go

```diff
@@ -43,7 +43,7 @@ func NewMapObjectEncoder() *MapObjectEncoder {
 
 // AddArray implements ObjectEncoder.
 func (m *MapObjectEncoder) AddArray(key string, v ArrayMarshaler) error {
-	arr := &sliceArrayEncoder{}
+	arr := &sliceArrayEncoder{elems: make([]interface{}, 0)}
```

Comment:
0 unnecessary in `make(..)` because it's the default.

Reply:
I initially tried it without the 0 and `zapcore/memory_encoder.go:46:39: missing len argument to make([]interface {})` popped up

Reply:
Derp, you're right.